examples/SingleParticleCurrent/include/simulation_defines/unitless/physicalConstants.unitless

```diff
@@ -97,4 +97,13 @@ namespace picongpu
     // = 1/c^2
     const float_X MUE0_EPS0 = float_X(1. / SPEED_OF_LIGHT / SPEED_OF_LIGHT);
 
+    /* Atomic unit of electric field in PIC Efield units */
+    const float_X ATOMIC_UNIT_EFIELD = float_X(SI::ATOMIC_UNIT_EFIELD / UNIT_EFIELD);
+
+    /* Atomic unit of time in PIC units */
+    const float_X ATOMIC_UNIT_TIME = float_X(SI::ATOMIC_UNIT_TIME / UNIT_TIME);
+
+    /* Time step of the simulation in atomic units */
+    const float_X DELTA_T_AU = float_X(SI::DELTA_T_SI / SI::ATOMIC_UNIT_TIME);
```

Comment:
`DELTA_T_AU`: I would calculate this value at the place where you need it. not here.